src/libs/kata-types/src/config/default.rs

```diff
@@ -14,6 +14,12 @@ use lazy_static::lazy_static;
 lazy_static! {
     /// Default configuration file paths, vendor may extend the list
     pub static ref DEFAULT_RUNTIME_CONFIGURATIONS: Vec::<&'static str> = vec![
+        // The rust runtime specific paths
```

Comment:
@jodh-intel Should the rust runtime looks for configs only under  dir "runtime-rs". I am not sure we need to keep both paths since the configs for golang and rust runtime are not compatible.

I know that the rust Makefile installs under `runtime-rs` directory, do we do the same for the configs as well?